modules/indexing/src/main/java/org/apache/ignite/internal/visor/query/VisorQueryDetailMetricsCollectorTask.java

```diff
@@ -96,13 +101,21 @@ protected VisorCacheQueryDetailMetricsCollectorJob(@Nullable VisorQueryDetailMet
          * @param since Time when metrics were collected last time.
          * @param res Response.
          * @param metrics Metrics.
+         * @param skipQueryMetrics Skip metrics for SQL and SQL_FIELD queries when {@code true}.
```

Comment:
Lets rename `skipQueryMetrics` to `collectNotSqlMetrics` and update javadoc accordingly.